src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotNetFormatTests.cs

```diff
@@ -47,19 +28,20 @@ public void VerifyNoChanges()
     [Fact]
     public void FormatProject()
     {
-        string testProjectPath = Path.Join(_formatTestsDirectoryPath, nameof(FormatProject));
-        string csprojPath = Path.Join(testProjectPath, nameof(FormatProject) + ".csproj");
+        string assetsDirectory = BaselineHelper.GetAssetsDirectory();
+
+        string unformattedCsFilePath = Path.Combine(assetsDirectory, TestFileName);
+        string solutionCsFilePath = Path.Combine(assetsDirectory, SolutionFileName);
```

Comment:
This name is a bit confusing to me.  To me something with `formatted` and `expected` more clearly describes the content .  e.g. `ExpectedFormattedCsFilePath`.  This applies to the actual file as well.